docs/loadtest_config.md

```diff
@@ -82,6 +82,12 @@ These will be created during the `init` process.
 
 The amount of active users to run when the load-test starts.
 
+### UsersFilePath
+
+*string*
+
+The path to the file which contains a list of user email and passwords that will be used by the tool if set. Each line should be for a user containing an email and password separated by space. It should have enough lines to contain the maximum number of users a load-test is expected to reach.
+
```

Comment:
Maybe we should also state that it shouldn't exceed the `MaxActiveUsers` limit.

Reply:
You mean it should be _at least_ `MaxActiveUsers` ;) It can exceed, that's not a problem.

Reply:
I see, yeah. Thanks for clarification 👍